gulpy/bch.py

```diff
@@ -8,8 +8,8 @@
 
 log = logging.getLogger(__name__)
 
-def process(data, sesh, diag):
-    successes, failures, skips = 0, 0, 0
+def process(data, sesh, diag, batch_size):
+    successes, failures, skips, batch = 0, 0, 0, 0
```

Comment:
Destructuring assignment should be used only where necessary to unpack a tuple or list. By contrast, this is just a harder to read shorthand for 4 simple assignments. Also, logically, `batch` is not in the same category as `successes`, `failures`, and `skips` but the single compound assignment sort of suggests it is.